build/dockerfiles/linux-libc.Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021-2022 Red Hat, Inc.
+# Copyright (c) 2021-2023 Red Hat, Inc.
```

Comment:
I don't think this change should be in the current PR

Reply:
well, this file has been updated many times in 2023 https://github.com/che-incubator/che-code/commits/main/build/dockerfiles/linux-libc.Dockerfile 

but you're right, it's inconsistent. I'll undo it here.

Reply:
```suggestion
# Copyright (c) 2021-2022 Red Hat, Inc.
```